lib/connections/service.go

```diff
@@ -641,3 +646,30 @@ func tlsTimedHandshake(tc *tls.Conn) error {
 	defer tc.SetDeadline(time.Time{})
 	return tc.Handshake()
 }
+
+func IsAllowedNetwork(host string, allowed []string) bool {
+	if len(allowed) == 0 {
```

Comment:
Maybe... Or I could move the len()==0 check out to the call site instead and make this a strict match only function?